src/queue/consumers/scicat/scicatProposal/consumerCallbacks/createChatroom.ts

```diff
@@ -50,6 +50,7 @@ const createChatroom = async (message: ValidProposalMessageData) => {
   const synapseService: SynapseService = container.resolve(
     Tokens.SynapseService
   );
+  await synapseService.login();
```

Comment:
question: what happens if you call synapseService.login() twice without calling logout() ? Will that still work?

I am just thinking what happens if error thrown from here until the try block is defined